docs/docs/genai/tracing/quickstart/index.mdx

```diff
@@ -0,0 +1,137 @@
+import ImageBox from '@site/src/components/ImageBox';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import TabsWrapper from '@site/src/components/TabsWrapper';
+import ServerSetup from "@site/src/content/setup_server.mdx";
+
+# Tracing Quickstart
+
+This quickstart guide will walk you through setting up a simple GenAI application with MLflow Tracing. In less than 10 minutes, you'll enable tracing, run a basic application, and explore the generated traces in the MLflow UI.
+
+## Prerequisites
+
+Make sure you have started the MLflow server.
+If you don't have the MLflow server running yet, just follow these simple steps to get it started.
+
+<ServerSetup />
+
+## Create a MLflow Experiment
+
+The traces your GenAI application will send to the MLflow server are grouped into MLflow experiments. We recommend creating one experiment for each GenAI application.
+
+Let's create a new MLflow experiment using the MLflow UI so that you can start sending your traces.
+
+<ImageBox src="/images/llms/tracing/quickstart/mlflow-ui-new-experiment.png" alt="New Experiment"/>
+
+1. Navigate to the MLflow UI in your browser at [http://localhost:5000](http://localhost:5000).
+2. Click on the <div className="inline-flex rounded-sm bg-sky-700 px-2 py-1.5 text-sm font-semibold text-white">Create</div> button on the top right.
+3. Enter a name for the experiment and click on "Create".
+
+_You can leave the `Artifact Location` field blank for now. It is an advanced configuration to override where MLflow stores experiment data._
```

Comment:
We may need more explanation on the artifact location, e.g. link to the guidance. But we can revisit this later since the current explanation seems too ML focused (https://mlflow.org/docs/latest/self-hosting/architecture/artifact-store/)